modules/statistics/RTPStatsCollector.js

```diff
@@ -275,61 +275,86 @@ StatsCollector.prototype._processAndEmitReport = function() {
         bitrateDownload += ssrcStats.bitrate.download;
         bitrateUpload += ssrcStats.bitrate.upload;
 
+        ssrcStats.resetBitrate();
```

Comment:
Why didn't we need this before?

Reply:
We needed this before also, but it was called at the end. I switched to using continue instead of using 5 nested if loops, therefore I need to call this before the first continue.

Reply:
👍